.idea/misc.xml

```diff
@@ -1,35 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is deleting the `.idea/misc.xml` from the `master` branch.  Is that your intention?

Reply:
Moving this conversation here:
> No, just from this PR ([link](https://github.com/frc1678/viewer-2018-android/pull/121#issuecomment-450531204))

Okay, currently this PR is requesting to delete the `.idea/misc.xml` file from `master`.  @KateUnger Could you update the PR by resetting the `.idea/misc.xml` to match master?